CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
 
+- [#574](https://github.com/nf-core/mag/pull/574) - Fix wrong channel going to BIN_SUMMARY (fix by @maxibor)
+
```

Comment:
That shouldnt be here, but ok.
But dont you want to add here that the full test was changed? Just to have it documented in the changelog...

Reply:
I realised it had been forgotten in the last PR... and made me forget my own entry indeed :laughing: